### PR TITLE
Feature: Add i18n - Telugu Support

### PR DESCRIPTION
**Pull request by Susmita-Dey**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature to add i18n - Telugu Support
- **Why this change was needed?** (You can also link to an open issue here)
Closes issue #1488 
- **Other information**:
Kindly review and merge this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The ultimate service for managing multi-channel notifications with a single API.
     <br />
 
   <br/>
-    <a href="https://github.com/novuhq/novu/issues">Report Bug</a>
+    <a href="https://github.com/novuhq/novu/issues/new?assignees=&labels=type%3A+bug&template=bug_report.yml&title=%F0%9F%90%9B+Bug+Report%3A+">Report Bug</a>
```
Comment (davidsoderberg) [inline README.md:35]: I think this comes from your other PR right?

Comment (Susmita-Dey) [reply README.md]: Deleted

Comment (davidsoderberg) [reply README.md]: You deleted the file?

Comment (Susmita-Dey) [reply README.md]: @davidsoderberg Sorry but one thing can be done is either I'll create a new PR or you can merge this PR and then merge #1487

File: packages/notification-center/src/i18n/languages/te.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const TE: ITranslationEntry = {
+  translations: {
+    notifications: 'నోటిఫికేషన్‌లు',
+    markAllAsRead: 'అన్నీచదివినట్లుగా సూచించు',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/te.ts:6]: Reversing with Google translate, seems not to be very accurate. Missing the `all` it seems.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,185 +0,0 @@
-<div align="center">
-  
-[![hacktoberfest-readme-dark](https://user-images.githubusercontent.com/63902456/193565711-96d0e494-998f-4f4b-aeb9-9aec80b24851.jpeg#gh-dark-mode-only)](https://novu.co/hacktoberfest#gh-dark-mode-only)
-
-</div>
```
[the remark this answers is not in the document]
Comment (Susmita-Dey) [reply README.md]: Added

File: packages/notification-center/src/i18n/languages/te.ts (hunk 2 above)
Comment (Susmita-Dey) [reply packages/notification-center/src/i18n/languages/te.ts:6]: No, it's there

**Comment by Susmita-Dey** [comment]
@davidsoderberg @p-fernandez Merge conflicts have been resolve. Kindly merge this PR.

**Pull request merged**